Library/Formula/ice.rb

```diff
@@ -35,7 +71,8 @@ def install
       embedded_runpath_prefix=#{prefix}
       OPTIMIZE=yes
     ]
-    args << "CXXFLAGS=#{ENV.cflags} -Wall -D_REENTRANT"
+
+    args << "CXXFLAGS=#{ENV.cflags} -Wall -D_REENTRANT -I#{libexec}/include -L#{libexec}/lib"
```

Comment:
You should instead set DB_HOME=#{libexec} in the args instead of manually adjusting the CXXFLAGS. That will ensure that the flags are only adjusted in those places that use BerkeleyDB instead of everywhere.